src/Yoti.Auth/DigitalIdentity/GetSessionResult.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Newtonsoft.Json;
+
+
+namespace Yoti.Auth.DigitalIdentity
+{
+    public class GetSessionResult
+    {
+#pragma warning disable 0649
+        // These fields are assigned to by JSON deserialization
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("expiry")]
+        public string Expiry { get; set; }
+
+        [JsonProperty("created")]
+        public string Created { get; set; }
+
+        [JsonProperty("updated")]
+        public string Updated { get; set; }
+
+        [JsonProperty("qrCode")]
+        public qrCode QrCode { get; set; }
+
+        [JsonProperty("receipt")]
+        public receipt Receipt { get; set; }
+
+#pragma warning restore 0649
+
+    }
+
+    public class qrCode
+    {
+        [JsonProperty("id")]
+        public string Id { get; set; }
+        [JsonProperty("uri")]
+        public string Uri { get; set; }
```

Comment:
minor: Even if it doesn't hurt mapping the `qrCode` URI may lead to confusion when an integrator finds a getter/setter in the API and no use for it.

Additionally the SDKs should be aligned as much as possible [[Java](https://github.com/getyoti/yoti-java-sdk/blob/master/yoti-sdk-api/src/main/java/com/yoti/api/client/identity/ShareSession.java), [Node](https://github.com/getyoti/yoti-node-sdk/blob/master/src/digital_identity_service/get.share.session.result.js)]

Reply:
You are right i have removed Uri to be align all sdk.